ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -832,6 +865,10 @@ const DataSetPage = fp.flow(withUserProfile(), withCurrentWorkspace(), withUrlPa
           case PrepackagedConceptSet.FITBITHEARTRATELEVEL:
             selectedPrePackagedConceptSDetEnum.push(PrePackagedConceptSetEnum.FITBITHEARTRATELEVEL);
             break;
+          case PrepackagedConceptSet.WHOLEGENOME:
+            selectedPrePackagedConceptSDetEnum.push(PrePackagedConceptSetEnum.WHOLEGENOME);
```

Comment:
nit: would be nice to fix this at the same time as fixing the FITIBIT typo

Reply:
Done